AppDB/appscale/datastore/cassandra_env/schema.py

```diff
@@ -248,6 +246,17 @@ def prime_cassandra(replication):
                   timeout=SCHEMA_CHANGE_TIMEOUT)
   session.set_keyspace(KEYSPACE)
 
+  logging.info('Waiting for all hosts to be connected')
+  deadline = time.time() + SCHEMA_CHANGE_TIMEOUT
+  while True:
+    if time.time() > deadline:
+      break
```

Comment:
@cdonati shouldn't we report warning in this case?
Without log message we have no straight way to verify if deadline was exceeded and we create table without having all nodes available.. it will look just like a normal creation of tables with all hosts online.

Reply:
@jeanleonov Yes, a warning log line would be appropriate. I'll update.